modules/virt-cnv-network-alerts.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-events.html/virt-virtualization-alerts.adoc
+
+[id="virt-cnv-network-alerts_{context}"]
+= Network alerts
+
+Network alerts provide information about problems for the CNV Network Operator.
+
+//KubeMacPoolDown Alert
+[id="KubeMacPoolDown_{context}"]
+== KubeMacPoolDown alert
+
+.Description
+
+The KubeMacPool is responsible for allocating MAC Addresses and preventing MAC Address conflicts.
+
+.Reason
+
+If the KubeMacPool-manager pod is down, then creating `VirtualMachine` objects fails.
+
+.Troubleshoot
+
+. Determine the Kubemacpool-manager pod namespace and name.
++
+[source,terminal]
+----
+export KMP_NAMESPACE="$(oc get pod -A --no-headers -l control-plane=mac-controller-manager | awk '{print $1}')"
+
+export KMP_NAME="$(oc get pod -A --no-headers -l control-plane=mac-controller-manager | awk '{print $2}')"
+----
+
+. Check the Kubemacpool-manager pod description and logs to determine the source of the problem.
++
+[source,terminal]
+----
+oc describe pod -n $KMP_NAMESPACE $KMP_NAME
```

Comment:
This applies to all lines that have the oc command syntax. The oc command should be preceded with $ and space:
$ oc describe pod...